crates/nu-command/src/system/run_external.rs

```diff
@@ -269,6 +269,14 @@ impl ExternalCommand {
                                         "'{}' was not found, did you mean '{s}'?",
                                         self.name.item
                                     )
+                                } else if self.name.item == s {
+                                    let sugg = engine_state.which_module_has_decl(s.as_bytes());
+                                    if let Some(sugg) = sugg {
+                                        let sugg = String::from_utf8_lossy(sugg);
+                                        format!("did you mean '{sugg} {s}'?")
```

Comment:
I'd rephrase it as "Command 'foo' was not found but it exists in a module 'bar'" or something like that. The "did you mean 'bar foo'?" suggestion might suggest there exists a command "bar foo" which is not true: You'd need to import it first.